Apps/phnetskope/netskope_run_query.html

```diff
@@ -12,7 +12,8 @@
 <!-- File: netskope_run_query.html
   Copyright (c) 2018-2019 Splunk Inc.
 
-  Licensed under Apache 2.0 (https://www.apache.org/licenses/LICENSE-2.0.txt)
+  SPLUNK CONFIDENTIAL - Use or disclosure of this material in whole or in part
+  without a valid written license from Splunk Inc. is PROHIBITED.
 -->
```

Comment:
Please revert. We now use an Apache license.

Reply:
Removed the license information.